src/components/ShareButton.tsx

```diff
@@ -48,9 +58,36 @@ function ShareButton({ text }: ShareButtonProps) {
       >
         {text}
         <Share className="margin-left-1" />
-      </Button>
+      </NoHoverButton>
     </span>
   );
 }
 
-export default ShareButton;
\ No newline at end of file
+const ShareButtonContainer = styled.div`
+  position: absolute;
+  right: 0;
+  top: 47%;
+  transform: translate(0%, -50%);
+  @media screen and (max-width: 45em) {
+    order: 1;
+    position: relative;
+    flex-basis: 100%;
+    transform: translate(0, 0);
+  }
+`;
+
+const ShareButtonContainerES = styled.div`
+  position: absolute;
+  right: 0;
+  top: 47%;
+  transform: translate(0%, -50%);
+  @media screen and (max-width: 65em) {
```

Comment:
nit: my hunch is that there's a DRY-er and more elegant way to write this if the only diff here is the max-width? but not a blocker